README_fr-FR.md

```diff
@@ -0,0 +1,443 @@
+
+Cela pourrait vous aider à comprendre ce qu'un commit est, pourquoi c'est important de bien écrire ses messages, les meilleurs techniques et quelques astuces, pour mieux (ré)écrire une bonne "commit history" 
+
+## Langages Disponibles
+
+- [English](README.md)
+- [Português](README_pt-BR.md)
+- [Deutsch](README_de-DE.md)
+- [Français](README_fr-FR.md)
+## Qu'est-ce qu'un "commit" ?
+
+Dans des termes simples, un commit est une _sauvegarde_ de vos fichiers local, écrits dans votre repository local.
+Contrairement à ce que certaines personnes pensent, [git ne stockent pas seulement la différence entre ces fichiers, il stock une version complète de tous les fichiers](https://git-scm.com/book/eo/v1/Ekkomenci-Git-Basics#Snapshots,-Not-Differences).
+For les fichiers qui ne changent pas d'un commit à l'autre, git stock just un lien vers la version précédente identique qui est déjà stockée.
+
+L'image ci-dessous montre comment git stock les fichiers à travers le temps, dans quelle "Version" est un commit.
+
+
+![](https://i.stack.imgur.com/AQ5TG.png)
+
+## Pourquoi les messages de commit sont importants ?
+
+- Pour raccourcir et faciliter les code-review
+- Pour aider dans la compréhension d'un changement.
+- Pour expliquer les "pourquoi", qui ne sauraient être décris en code
+- Pour aider les futurs personnes qui vont devoir maintenir le code à comprendre les différents changements, et l'évolution du code.
+
+Pour maximiser ces résultats, nous pouvons utiliser certaines bonnes pratiques et des standards, décrits dans la prochaine section.
+
+## Bons Usages
+
+Il y a quelques bons usages recueillit par l'experience, des articles sur internet, et d'autres guides. Si vous en avez d'autres (ou n'êtes pas d'accord avec certains), soyez libres d'ouvrir une demande de Pull et de contribuer à l'amélioration du guide.
+
+### Utilisez l'impératif
+
+```
+# Bon
+Use InventoryBackendPool to retrieve inventory backend
+```
+
+```
+# Mauvais 
+Used InventoryBackendPool to retrieve inventory backend
+```
+
+_Pourquoi l'impératif ?__
+
+Un message de commit décrit ce que le changement **fait**, ces effets, pas ce qui à été fait.
+
+[Cet excelent article de Chris Beams](https://chris.beams.io/posts/git-commit/)
+Nous donne une phrase simple qui peut être utilisée pour mieux écrire les messages de commit à l'impératif.
+
+```
+If applied, this commit will <commit message>
+```
+
+Examples:
+
+```
+# Bon
+If applied, this commit will use InventoryBackendPool to retrieve inventory backend
+```
+
+```
+# Mauvais
+If applied, this commit will used InventoryBackendPool to retrieve inventory backend
+```
+
+### Capitalize the first letter
+
+```
+# Bon
+Add `use` method to Credit model
+```
+
+```
+# Mauvais
+add `use` method to Credit model
+```
+
+The reason that the first letter should be capitalized is to follow the grammar rule of using capital letters at the beginning of sentences.
```

Comment:
`The reason that the first letter should be capitalized is to follow the grammar rule of using capital letters at the beginning of sentences.` > `La première lettre doit être en majuscule pour suivre la règle de grammaire qui consiste à utiliser des majuscules au début des phrases.`